### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by dubstard**
<html>
<body>

Site | URLScan link
-- | --
accessbot.net|https://urlscan.io/result/92577b90-fb12-4bba-9f0a-f75fc51e0e86/dom/
anchorprotocolapp.info|https://user-images.githubusercontent.com/49607867/167994770-16382efd-32a7-49c5-ad1c-de95acd38606.png
apinetworkbridge.net|https://urlscan.io/result/0a19f822-3d27-4a54-9862-c96dd6b2d742/dom/
chainnodedeeplink-bridge.online|https://user-images.githubusercontent.com/49607867/168009568-16f2a088-adc2-4219-a2f9-e64e05862d75.png
coin.swiftconnectdapp.com|https://urlscan.io/result/05f08fe1-3fc4-4535-a306-819976c6299c/dom/
col1ab.land|https://user-images.githubusercontent.com/49607867/167990342-c4b41c11-66e1-4237-a7df-155d349b110a.png
mainnetaccess.com|https://urlscan.io/result/1a08eaac-0c25-4452-bd88-fef01a4c9e8b/dom/
safenfts.xyz|https://urlscan.io/result/af347820-79a0-47ac-99b9-c7292ee004cf/dom/
solutionandsync.org|https://urlscan.io/result/7a5022d0-802c-4bcb-bcda-23312ec8fb2c/dom/
swiftconnectdapp.com|https://user-images.githubusercontent.com/49607867/168009375-2c688549-90d2-47ff-8790-dc9f79d81f4e.png
tokenerror-resolution.online|https://urlscan.io/result/3f868836-2d64-490e-b3c4-42c7196f4186/dom/
tokenrequestreward.online|https://urlscan.io/result/e67f155c-1473-4c5c-9310-971d444de364/dom/
tokentpocket.pro|https://user-images.githubusercontent.com/49607867/168009711-13b826fb-2345-4820-bd95-e8b810cbc20d.png
trconnect.online|https://urlscan.io/result/b36cb25b-a55b-47da-ac14-33bd6541509f/dom/

</body>
</html>

"accessbot.net",
"anchorprotocolapp.info",
"apinetworkbridge.net",
"col1ab.land",
"kcoins.exchange",
"mainnetaccess.com",
"poilegon-technology.org",
"protocol-verify.info", - Geofenced, redirects to trustwallet unless you are on a mobile device and ANZ IP?
"safenfts.xyz",
"solutionandsync.org",
"stepen-code.online",
"trconnect.online",

### CODE REVIEW

**Review by dubstard — COMMENTED** [review]
remove duplicate
  "mainnetaccess.com",

File: all.json [hunk 1]
```diff
@@ -14,6 +14,17 @@
 		"1inch-network.net",
 		"1inch-wallet.org",
 		"1inch.tk",
+		"accessbot.net",
+		"anchorprotocolapp.info",
+		"apinetworkbridge.net",
+		"col1ab.land",
+		"kcoins.exchange",
+		"poilegon-technology.org",
```
Comment (jacogr) [inline all.json:22]: Not sure I get this one? Food-related site?

Comment (dubstard) [reply all.json:22]: It is a clear imitation of polygon.

fake misspelled
`poilegon-technology.org`
real
`polygon.technology`

You either need to have a specific finger print (i.e. mobile device) and access it from a given geographical location in order to get the phishing content (i think it is in /assets/)

You need to fuzz it to get to the phish and it is not worth the effort. Block on sight is the best. 

Scam sites would often display random junk content, before they get weaponized, or a scam landing page.
If you don't know where the phishing is you may miss it.

This is done to confuse automated scanners, parsers and threat annalists.

image
image

**Comment by dubstard** [comment]
WIP, adding more

**Review by dubstard — COMMENTED** [review]
add
		"app-bestdappfixes.online",
		"app.restoreassets.org",
		"chainnodedeeplink-bridge.online",
		"coin.swiftconnectdapp.com",
		"swiftconnectdapp.com",
		"dapp-eth.expert",
		"fixdapps.online",
		"imtoken-7q.xyz",
		"imtoken-rw.xyz",
		"imtoken-sd.xyz",
		"juzztoken.com",
		"nodedappsnet.com",
		"recovery-verify-metamask.com",
		"restoreassets.org",
		"synchronisedcerrorfix.com",
		"tokenerror-resolution.online",
		"tokenrequestreward.online",
		"tokentpocket.pro",
		"unlock-wallet-connect.com",

**Review by dubstard — COMMENTED** [review]
get rid of `poilegon-technology.org`

**Comment by dubstard** [comment]
✅Ready

**Review by frankywild — APPROVED** [review]

**Comment by dubstard** [comment]
Thanks, Franky!

**Pull request merged**